frontend/packages/data-portal/app/components/Run/RunHeader.tsx

```diff
@@ -88,78 +89,87 @@ export function RunHeader() {
       ]}
       onMoreInfoClick={() => drawer.setActiveDrawerId('run-metadata')}
       title={run.name}
-    >
-      <div className="flex gap-sds-xxl p-sds-xl border-t-[3px] border-sds-gray-200">
-        <div className="max-w-[300px] max-h-[213px] grow overflow-clip rounded-sds-m flex-shrink-0">
-          {keyPhotoURL ? (
-            <Link to={keyPhotoURL}>
-              <KeyPhoto title={run.name} src={keyPhotoURL} />
-            </Link>
-          ) : (
-            <KeyPhoto title={run.name} />
-          )}
-        </div>
+      renderHeader={({ moreInfo }) => (
+        <div className="flex gap-sds-xxl p-sds-xl border-t-[3px] border-sds-gray-200">
+          <div className="max-w-[300px] max-h-[213px] grow overflow-clip rounded-sds-m flex-shrink-0">
+            {keyPhotoURL ? (
+              <Link to={keyPhotoURL}>
+                <KeyPhoto title={run.name} src={keyPhotoURL} />
+              </Link>
+            ) : (
+              <KeyPhoto title={run.name} />
+            )}
+          </div>
+
+          <div className="flex flex-col gap-sds-xl">
+            <div className="flex flex-1 gap-sds-xxl flex-col lg:flex-row">
```

Comment:
it seems github didnt actually save it but my comment was to move `flex-1` up a div since in your screenshot, the key photo height for the single runs page doesnt match that of the table + button as in the design